### PR TITLE
Adds MBS forms 0203, 0204 and 0253

### PR DESCRIPTION
**Pull request by nealedj**

### CODE REVIEW

File: data/en/mbs_0203.json
Comment (MIslam92) [inline data/en/mbs_0203.json:9]: eq_id and form_type need to be added here

File: data/en/mbs_0204.json
Comment (MIslam92) [inline data/en/mbs_0204.json:10]: eq_id and form_type need to be added here

**Review by MIslam92 — DISMISSED** [review]

File: data/en/mbs_0203.json
Comment (nealedj) [reply data/en/mbs_0203.json:9]: We'll hold off on this as ideally we shouldn't need to add this to all our schemas. Most schemas don't have it.

If it becomes a problem for previewing then we'll add to all of them.

File: data/en/mbs_0203.json
Comment (dwyeradam) [inline data/en/mbs_0203.json]: I don't think we need the `(megalitres)` here. Not on the slide or block title.

File: data/en/mbs_0203.json
Comment (dwyeradam) [inline data/en/mbs_0203.json]: `potable water that was supplied to customers` should be highlighted (`<em>`)

Comment (dwyeradam) [reply data/en/mbs_0203.json]: I also think the question title should be the block title and the question description should be the question title.

**Review by dwyeradam — CHANGES_REQUESTED** [review]
Initial review of 0203, i'm assuming the same issues will be present for the others.

File: data/en/mbs_0253.json
Comment (dwyeradam) [inline data/en/mbs_0253.json]: Should go to Significant Changes block.

**Review by dwyeradam — CHANGES_REQUESTED** [review]
Same comments as 0203 with an additional routing issue.

File: data/en/mbs_0203.json
Comment (shahi645) [inline data/en/mbs_0203.json:27]: Its like this in the spec, but should it not have NI?

File: data/en/mbs_0203.json
Comment (shahi645) [inline data/en/mbs_0203.json:213]: This schema LGTM

File: data/en/mbs_0204.json
Comment (shahi645) [inline data/en/mbs_0204.json:27]: Think this should just say NI?

**Comment by shahi645** [comment]
@nealedj Think you have other commits in here from feature branch, but rest of it LGTM, so can approve when this is sorted

**Review by dwyeradam — APPROVED** [review]
The three (0203, 0204 and 0253) schemas look good to me.

File: data/en/mbs_0203.json
Comment (charlotte-hirst) [inline data/en/mbs_0203.json]: its not like this in slides and I think QBS is the same where there is only 1 question so doesnt use twistees and looks really weird as nothing in it, i would just add as a line of text so less weird

File: data/en/mbs_0203.json
Comment (charlotte-hirst) [inline data/en/mbs_0203.json:125]: font size isnt right the question is too small a font
image

File: data/en/mbs_0204.json
Comment (charlotte-hirst) [inline data/en/mbs_0204.json]: same thing i would lose twistee

**Comment by charlotte-hirst** [comment]
footer is broken on 204 
image

File: data/en/mbs_0204.json
Comment (charlotte-hirst) [inline data/en/mbs_0204.json:124]: same as 203 wrong font sizes

File: data/en/mbs_0204.json
Comment (charlotte-hirst) [inline data/en/mbs_0204.json:213]: broken footer across form type

File: data/en/mbs_0253.json
Comment (charlotte-hirst) [inline data/en/mbs_0253.json:179]: again weird font size

File: data/en/mbs_0253.json
Comment (charlotte-hirst) [inline data/en/mbs_0253.json:281]: Question is wrong, slides are correct
Of the 5 total employees employed on 10 June 2016, please specify the number of:
slides
Of the {{employee total figure}} total employees employed on {{employment_date}}, how many male and female employees worked the following hours?

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

File: data/en/mbs_0203.json
Comment (nealedj) [reply data/en/mbs_0203.json:125]: Looks like that's a little out of date. I corrected that yesterday afternoon.

File: data/en/mbs_0204.json
Comment (nealedj) [reply data/en/mbs_0204.json:213]: I'm not seeing that. It'll be something to do with the NI theme not rendering correctly for you. We'll figure out what's going on locally there.

File: data/en/mbs_0203.json
Comment (nealedj) [reply data/en/mbs_0203.json]: I've put it into a single item list area that we already have in the pattern and removed it from the accordion.

**Review by charlotte-hirst — APPROVED** [review]
LGTM

**Review by shahi645 — APPROVED** [review]

**Pull request merged**
